pulp_rpm/app/models/modulemd.py

```diff
@@ -92,3 +97,79 @@ def natural_key_fields(cls):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+
+
+class ModulemdObsolete(Content):
+    """
+    The "Modulemd Obsoletes" content type.
+
+    Fields:
+        modified (String):
+            A string representing UTC date in ISO 8601 format: YYYY-MM-DD[T ]HH:MMZ
+        reset (Bool):
+            A boolean option to reset all previously specified obsoletes
+        module (String):
+            A string representing a Name of a module that is EOLed
+        stream (String):
+            A string representing a Stream of a module that is EOLed
+        context (String):
+            A string representing a Context of a module that is EOLed
+        eol_date (String):
+            A string representing UTC date in ISO 8601 format: YYYY-MM-DD[T ]HH:MMZ
+        message (String):
+            A string describing the change, reason, etc.
+        obsolete_by (JSON):
+            A dict to provide details about the obsoleting module and stream
+    """
+
+    TYPE = "modulemd_obsolete"
+
+    # Mandatory fields
+    # we need to store a modulemd obsoletes document version
+    # because we need it to know in publish time
```

Comment:
Do we?  I don't see it used in the publish code, and I wouldn't think we need it

Reply:
We don't. It is leftover from no snippet implementation.